themes/ddbasic/templates/node/node--view-mode--search-result.tpl.php

```diff
@@ -82,5 +82,6 @@
   <div class="content"<?php print $content_attributes; ?>>
     <a href="<?php print $node_url; ?>"><h3 class="title"><?php print $title; ?></h3></a>
     <?php print render($content); ?>
+    <a href="<?php print $node_url; ?>"><div class="button"><?php print t('Read more'); ?></div></a>  
```

Comment:
Any reason why we print the link manually instead of using [`l()`](https://api.drupal.org/api/drupal/includes%21common.inc/function/l/7.x)?